doc/classes/RandomNumberGenerator.xml

```diff
@@ -5,6 +5,14 @@
 	</brief_description>
 	<description>
 		RandomNumberGenerator is a class for generating pseudo-random numbers. It currently uses PCG32. The underlying algorithm is an implementation detail. As a result, it should not be depended upon for reproducible random streams across Godot versions.
+		To generate a random float number (within a given range) based on a time-dependant seed:
+		[codeblock]
+		const MIN_RANGE = float(-10)
+		const MAX_RANGE = float(10)
+		var random_number = RandomNumberGenerator.new()
+		random_number.randomize()
```

Comment:
I think this may be a bit misleading, considering that your code example is apparently targeting absolute beginners. They might think it's necessary to make a new RNG instance / call randomize every time they need a new number (which will ruin the distribution). I'd suggest putting it into context by using it in an actual node:

```
var rng
func _ready():
    rng = RandomNumberGenerator.new()
    rng.randomize()
func _process(_delta):
    var random_number = rng.randf_range(-10.0, 10.0)
    # ...do stuff with random_number...
```

Reply:
Thanks for the suggestion

Reply:
@tree786 If you agree with the suggestion, could you please update your PR? And in case you disagree with it, could you state your reasons so others may see them? A dry "thanks for the suggestion" statement is uninformative to the rest of the community.

Reply:
Yes, please wait a bit. I agree with what you said, but I ran into issues with updating the PR. I will update it with the changes mentioned in this thread. Bear in mind I'm still practicing with GitHub (as I hinted at earlier).

Reply:
There is a section about updating your PR in the docs. https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#updating-your-branch

And why are the conversations marked as resolved? They're not resolved because you haven't made the changes yet.